src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationOptions.cs

```diff
@@ -25,153 +26,21 @@ public LinkedInAuthenticationOptions()
         UserInformationEndpoint = LinkedInAuthenticationDefaults.UserInformationEndpoint;
         EmailAddressEndpoint = LinkedInAuthenticationDefaults.EmailAddressEndpoint;
 
-        Scope.Add("r_liteprofile");
-        Scope.Add("r_emailaddress");
+        Scope.Add("openid");
+        Scope.Add("profile");
+        Scope.Add("email");
 
         ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, ProfileFields.Id);
-        ClaimActions.MapCustomJson(ClaimTypes.Name, user => GetFullName(user));
-        ClaimActions.MapCustomJson(ClaimTypes.GivenName, user => GetMultiLocaleString(user, ProfileFields.FirstName));
-        ClaimActions.MapCustomJson(ClaimTypes.Surname, user => GetMultiLocaleString(user, ProfileFields.LastName));
-        ClaimActions.MapCustomJson(Claims.PictureUrl, user => GetPictureUrls(user)?.LastOrDefault());
-        ClaimActions.MapCustomJson(Claims.PictureUrls, user =>
-        {
-            var urls = GetPictureUrls(user);
-            return urls == null ? null : string.Join(',', urls);
-        });
+        ClaimActions.MapJsonKey(ClaimTypes.Name, ProfileFields.Name);
+        ClaimActions.MapJsonKey(ClaimTypes.Email, ProfileFields.Email);
+        ClaimActions.MapJsonKey(ClaimTypes.GivenName, ProfileFields.GivenName);
+        ClaimActions.MapJsonKey(ClaimTypes.Surname, ProfileFields.FamilyName);
+        ClaimActions.MapJsonKey(Claims.Picture, Claims.Picture);
+        ClaimActions.MapJsonKey(Claims.EmailVerified, Claims.EmailVerified);
     }
 
     /// <summary>
     /// Gets or sets the email address endpoint.
     /// </summary>
     public string EmailAddressEndpoint { get; set; }
-
-    /// <summary>
-    /// Gets the list of fields to retrieve from the user information endpoint.
-    /// See <a>https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin</a> for more information.
-    /// </summary>
-    public ISet<string> Fields { get; } = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
-        {
-            ProfileFields.Id,
-            ProfileFields.FirstName,
-            ProfileFields.LastName,
-            EmailAddressField
-        };
-
-    /// <summary>
-    /// Gets or sets a <c>MultiLocaleString</c> resolver, a function which takes all localized values
-    /// and an eventual preferred locale from the member and returns the selected localized value.
-    /// The default implementation resolve it in this order:
-    /// 1. Returns the <c>preferredLocale</c> value if it is set and has a value.
-    /// 2. Returns the value corresponding to the <see cref="Thread.CurrentUICulture"/> if it exists.
-    /// 3. Returns the first value.
-    /// </summary>
-    /// <see cref="DefaultMultiLocaleStringResolver(IReadOnlyDictionary{string, string}, string?)"/>
-    public Func<IReadOnlyDictionary<string, string?>, string?, string?> MultiLocaleStringResolver { get; set; } = DefaultMultiLocaleStringResolver;
```

Comment:
As these are public, removing them is a breaking change - they should be marked as obsolete instead.

We should also retain the private `DefaultMultiLocaleStringResolver` method, as otherwise it's a breaking change of the behaviour of the `MultiLocaleStringResolver` property if someone is using it for another purpose.

Reply:
Implemented the changes as advised